bin/test-code-style

```diff
@@ -5,7 +5,8 @@
 runSbt  scalariformFormat \
         test:scalariformFormat \
         multi-jvm:scalariformFormat \
-        checkHeaders \
+        headerCreate \
+        headerCheck \
         test:checkHeaders \
```

Comment:
`test:checkHeader` -> `test:headerCheck`

`multi-jvm:checkHeaders` -> `multi-jvm:headerCheck`

Reply:
Thanks, @marcospereira. Do you know if I need to do the same for `headerCreate`? i.e., create the headers in various settings.

Reply:
Hi @erip,

No, you don't need the `headerCreate`. It was probably used before to change the files and then check with git if there are differences. `headerCheck` will do the job now and we only need the `git diff` to check the code format.